cpu/stm32l4/vectors.c

```diff
@@ -18,236 +18,185 @@
-    isr_i2c3_ev,            /* [72] I2C3 event interrupt */
-    isr_i2c3_er,            /* [73] I2C3 error interrupt */
-    isr_sai1,               /* [74] Serial Audio Interface 1 global interrupt */
-    isr_sai2,               /* [75] Serial Audio Interface 2 global interrupt */
-    isr_swpmi1,             /* [76] Serial Wire Interface 1 global interrupt */
-    isr_tsc,                /* [77] Touch Sense Controller global interrupt */
-    isr_lcd,                /* [78] LCD global interrupt */
-    (0UL),                  /* [79] Reserved*/
-    isr_rng,                /* [80] RNG global interrupt */
-    isr_fpu                 /* [81] FPU global interrupt */
+    [18] = isr_adc1,                 /* [18] ADC1 global Interrupt */
+    [26] = isr_tim1_trg_com,         /* [26] TIM1 Trigger and Commutation Interrupt */
+    [67] = isr_usb,                  /* [67] USB event Interrupt */
+    [82] = isr_crs,                  /* [82] CRS global interrupt */
+#elif defined(CPU_MODEL_STM32L476RG)
+    [18] = isr_adc1_2,               /* [18] ADC1, ADC2 SAR global Interrupts */
+    [26] = isr_tim1_trg_com_tim17,   /* [26] TIM1 Trigger and Commutation Interrupt and TIM17 global interrupt */
+    [29] = isr_tim3,                 /* [29] TIM3 global Interrupt */
+    [30] = isr_tim4,                 /* [30] TIM4 global Interrupt */
+    [33] = isr_i2c2_ev,              /* [33] I2C2 Event Interrupt */
+    [34] = isr_i2c2_er,              /* [34] I2C2 Error Interrupt */
+    [36] = isr_spi2,                 /* [36] SPI2 global Interrupt */
+    [39] = isr_usart3,               /* [39] USART3 global Interrupt */
+    [42] = isr_dfsdm1_flt3,          /* [42] DFSDM1 Filter 3 global Interrupt */
+    [43] = isr_tim8_brk,             /* [43] TIM8 Break Interrupt */
+    [44] = isr_tim8_up,              /* [44] TIM8 Update Interrupt */
+    [45] = isr_tim8_trg_com,         /* [45] TIM8 Trigger and Commutation Interrupt */
+    [46] = isr_tim8_cc,              /* [46] TIM8 Capture Compare Interrupt */
+    [47] = isr_adc3,                 /* [47] ADC3 global  Interrupt */
+    [48] = isr_fmc,                  /* [48] FMC global Interrupt */
+    [49] = isr_sdmmc1,               /* [49] SDMMC1 global Interrupt */
+    [50] = isr_tim5,                 /* [50] TIM5 global Interrupt */
+    [52] = isr_uart4,                /* [52] UART4 global Interrupt */
+    [53] = isr_uart5,                /* [53] UART5 global Interrupt */
+    [61] = isr_dfsdm1_flt0,          /* [61] DFSDM1 Filter 0 global Interrupt */
+    [62] = isr_dfsdm1_flt1,          /* [62] DFSDM1 Filter 1 global Interrupt */
+    [63] = isr_dfsdm1_flt2,          /* [63] DFSDM1 Filter 2 global Interrupt */
+    [67] = isr_otg_fs,               /* [67] USB OTG FS global Interrupt */
+    [75] = isr_sai2,                 /* [75] Serial Audio Interface 2 global interrupt */
+    [78] = isr_lcd,                  /* [78] LCD global interrupt */
```

Comment:
2 interrupts are missing for stm32l476rg:
```
[80] = isr_rng,
[81] = isr_fpu,
```

Reply:
hm, thats bad. Need to look into this

Reply:
Looked into this: they are not missing. Both vectors are defined in the common section above.